forms.py

```diff
@@ -1,14 +1,28 @@
 from flask_wtf import FlaskForm
-from wtforms import StringField, PasswordField
-from wtforms.validators import InputRequired, Email
+from wtforms import StringField, PasswordField, SelectField
+from wtforms.validators import InputRequired, Email, Optional
+
+units = ["Unit 1", "Unit 2", "Unit 3", "Unit 4", "Unit 5", "Unit 6", "Unit 7",
```

Comment:
May consider moving this into a model that has a similar but reduced API as a orm model